# fake params using in fake driver
ned2:
    id_list: [2, 3, 4, 5, 6, 7, 8, 11]
    steppers:
      id: [2, 3, 4, 8]
      position: [0, 0, 0, 0]
      velocity: [0, 0, 0, 0]
      temperature: [50, 52, 54, 55]
      voltage: [12.1, 12.2, 12.3, 12]
      min_position: [0, 0, 0, 0]
      max_position: [4096, 4096, 4096, 4096]
      model_number: [1, 1, 1, 2]
      firmware: ["0.0.1", "0.0.01", "0.0.1", "0.0.1"]
    dynamixels:
      id: [5, 6, 7, 11]
      position: [2048, 2048, 2048, 370]
      velocity: [0, 0, 0, 0]
      temperature: [50, 52, 54, 56]
      voltage: [50, 50, 50, 50]
      min_position: [0, 0, 0, 0]
      max_position: [4096, 4096, 4096, 4096]
      model_number: [1, 1, 1, 1]
      firmware: ["0.0.1", "0.0.01", "0.0.1", "0.0.1"]
    end_effector:
      id: 0
      temperature: 32
      voltage: 5000
      firmware: "0.0.1"
ned:
    id_list: [2, 3, 6, 11]
    dynamixels:
      id: [2, 3, 6, 11]
      position: [2048, 2048, 2048, 370]
      velocity: [0, 0, 0, 0]
      temperature: [50, 52, 54, 56]
      voltage: [50, 50, 50, 50]
      min_position: [0, 0, 0, 0]
      max_position: [4096, 4096, 4096, 4096]
      model_number: [1, 1, 1, 1]
      firmware: ["0.0.1", "0.0.01", "0.0.1", "0.0.1"]

one:
    id_list: [2, 3, 6, 11]
    dynamixels:
      id: [2, 3, 6, 11]
      position: [2048, 2048, 2048, 370]
      velocity: [0, 0, 0, 0]
      temperature: [50, 52, 54, 56]
      voltage: [50, 50, 50, 50]
      min_position: [0, 0, 0, 0]
      max_position: [4096, 4096, 4096, 4096]
      model_number: [1, 1, 1, 1]
      firmware: ["0.0.1", "0.0.01", "0.0.1", "0.0.1"]
